### ISSUE TITLE
BCarousel unimplemented props

### ISSUE THREAD

**Issue by dwgray** [bootstrap-vue-next/bootstrap-vue-next#2180]
### Missing feature is from

bootstrap-vue

### Docs

https://bootstrap-vue.org/docs/components/carousel#comp-ref-b-carousel-props

### Describe the missing feature

`label-indicators` sets the `aria-label` on the indicator control wrappers - this is missing and I don't see a different way to do that
`no-animation` disables the animation when transitioning between slides - I don't see any other way to do this.

### Validations

- [X] Have tested with the latest version. This is still alpha version and sometime things change rapidly.
- [X] Follow our [Code of Conduct](https://github.com/bootstrap-vue-next/bootstrap-vue-next/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/bootstrap-vue-next/bootstrap-vue-next/blob/main/CONTRIBUTING.md).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [X] The provided link is to the documentation of the missing feature.

**Comment by xvaara** [comment]
isn't indicatorsButtonLabel prop setting the label? or is that doing something else in BV2?

I'll add the no-animation prop

**Comment by dwgray** [comment]
@xvaara - `indicators-button-label` sets the aria prop on the individual buttons. In BSV `label-indicators` sets the `aria-label` of the element that contains those controls - the default is "Select a slide to display" - this seems like it would be useful for someone depending on a screen reader. I would completely support a different name for this prop though. Maybe `indicator-container-label`?

**Comment by loicduong** [comment]
Resolved in https://github.com/bootstrap-vue-next/bootstrap-vue-next/pull/2293 and still matches BV2 is `label-indicators`
